mantle/kola/harness.go

```diff
@@ -533,6 +538,58 @@ func metadataFromTestBinary(executable string) (*externalTestMeta, error) {
 	return meta, nil
 }
 
+func runExternalTest(c cluster.TestCluster, mach platform.Machine) error {
+	var previousRebootState string
+	var stdout []byte
+	var stderr []byte
+	for {
+		bootID, err := platform.GetMachineBootId(mach)
+		if err != nil {
+			return errors.Wrapf(err, "getting boot id")
+		}
+		plog.Debug("Starting kolet run-test-unit")
+		if previousRebootState != "" {
+			plog.Debugf("Setting AUTOPKGTEST_REBOOT_MARK=%s", previousRebootState)
+			c.MustSSHf(mach, "echo AUTOPKGTEST_REBOOT_MARK=%s | sudo tee /run/kola-runext-env", previousRebootState)
+			previousRebootState = ""
+		}
+		// We ignore stderr here, it will end up in the journal
```

Comment:
This comment says we ignore stderr, but we do capture it and print it on failure, no?